addons/binding/org.openhab.binding.dwdunwetter/src/main/java/org/openhab/binding/dwdunwetter/internal/DwdUnwetterConfiguration.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.dwdunwetter.internal;
+
+import org.apache.commons.lang.StringUtils;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * The {@link DwdUnwetterConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Martin Koehler - Initial contribution
+ */
+public class DwdUnwetterConfiguration {
+
+    private @Nullable Integer refresh;
```

Comment:
Please remove the `@Nullable` annotations. Rather add `NonNullByDefault` on class level. Configuration parameters which are required or define a default value will never be `null`. Thus you additionally can change them to primitive types like e.g. `int`.

Reply:
Done